spynnaker_integration_tests/test_various/test_alltoone_with_large_weight.py

```diff
@@ -40,7 +41,10 @@ def do_run(self):
         sim.end()
 
         weight_sum = sum(weight[2] for weight in weight_list)
-        self.assertEqual(weight_sum, sources * weights)
+        # 50.0 is not exactly representable so work out the actual value
+        weight_used = 1 / (
```

Comment:
Would AlmostEqual not work even without this adjustment.

I would prefer that as then you are testing without the same adjustment as in the code.

Reply:
It would work - I'd just need to work out the correct tolerance for it...

Reply:
Done now.